.github/workflows/sync-docs.yml

```
@@ -0,0 +1,71 @@
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get changed and removed files
+        id: files
+        run: |
+          git fetch origin ${{ github.event.before }}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)